### PR TITLE
Fix whitespace lint

### PR DESCRIPTION
**Pull request by Phantsure**
Update lint workflow. This workflow would report extra white spaces in different files

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Lint
 
 on:
-  pull_request:
+  pull_request_target:
```
Comment (anuragc617) [inline .github/workflows/lint.yaml:4]: why do we need this to be pull_request_target?

Comment (Phantsure) [reply .github/workflows/lint.yaml:4]: So that they don't require approval for seeing linting  issue

Comment (anuragc617) [reply .github/workflows/lint.yaml:4]: For forked repo PRs, workflows with `pull_request` get repo `read` permission while workflows triggered with `pull_request_target` get repo `read/write`  permission. The `pull_request_target` should not be used  for workflows that can commit the code repo.  Is this workflow only points out linting issues or it fixes them as well via code commit? If it is pushing code then we should not use `pull_request_target`.

Comment (Phantsure) [reply .github/workflows/lint.yaml:4]: It fixes them as well but doesn't commit or push the changes. Changes are made to local runner git repo

Comment (Phantsure) [reply .github/workflows/lint.yaml]: Have made it back to `pull_request`

Comment (Phantsure) [reply .github/workflows/lint.yaml:4]: back to `pull_request_target`

**Pull request merged**